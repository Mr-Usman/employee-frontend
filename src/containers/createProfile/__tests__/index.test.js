import React from "react";
import CreateProfile from "../index";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

describe("renders <CreateProfile /> Component", () => {
  it("match the snapshot", () => {
    const tree = renderer.create(<CreateProfile />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
