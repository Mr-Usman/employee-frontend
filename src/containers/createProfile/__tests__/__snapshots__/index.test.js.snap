// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <CreateProfile /> Component match the snapshot 1`] = `
<div
  className="sc-AxjAm bcThxe"
>
  <h1
    className="sc-AxirZ kHuMvb"
  >
    Create a New User
  </h1>
  <form
    className=""
    onSubmit={[Function]}
  >
    <div
      className="row"
    >
      <div
        className="col-md-2"
      />
      <div
        className="col-md-4"
      >
        <div
          className="form-group"
        >
          <label
            className="form-label"
          >
            Email
          </label>
          <input
            className="form-control"
            name="email"
            onChange={[Function]}
            type="email"
            value=""
          />
        </div>
      </div>
      <div
        className="col-md-4"
      >
        <div
          className="form-group"
        >
          <label
            className="form-label"
          >
            Password
          </label>
          <input
            className="form-control"
            name="password"
            onChange={[Function]}
            type="password"
            value=""
          />
        </div>
      </div>
      <div
        className="col-md-2"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-2"
      />
      <div
        className="col-md-4"
      >
        <div
          className="form-group"
        >
          <label
            className="form-label"
          >
            Select Role
          </label>
          <select
            className="form-control"
            onChange={[Function]}
          >
            <option>
              role
            </option>
            <option
              value="admin"
            >
              admin
            </option>
            <option
              value="manager"
            >
              manager
            </option>
            <option
              value="developer"
            >
              developer
            </option>
          </select>
        </div>
      </div>
      <div
        className="col-md-4"
      >
        <div
          className="form-group"
        >
          <label
            className="form-label"
          >
            Permissions
          </label>
          <div
            className="form-group"
          >
            <div
              className="form-check form-check-inline"
            >
              <input
                className="form-check-input"
                disabled={false}
                name="create"
                onChange={[Function]}
                type="checkbox"
              />
              <label
                className="form-check-label"
                title=""
              >
                Create
              </label>
            </div>
            <div
              className="form-check form-check-inline"
            >
              <input
                className="form-check-input"
                disabled={false}
                name="read"
                onChange={[Function]}
                type="checkbox"
              />
              <label
                className="form-check-label"
                title=""
              >
                Read
              </label>
            </div>
            <div
              className="form-check form-check-inline"
            >
              <input
                className="form-check-input"
                disabled={false}
                name="update"
                onChange={[Function]}
                type="checkbox"
              />
              <label
                className="form-check-label"
                title=""
              >
                Update
              </label>
            </div>
            <div
              className="form-check form-check-inline"
            >
              <input
                className="form-check-input"
                disabled={false}
                name="delete"
                onChange={[Function]}
                type="checkbox"
              />
              <label
                className="form-check-label"
                title=""
              >
                Delete
              </label>
            </div>
          </div>
        </div>
      </div>
      <div
        className="col-md-2"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-2"
      />
      <div
        className="col-md-8"
      >
        <div
          className="form-group"
        >
          <button
            className="btn btn-primary"
            disabled={false}
            type="submit"
          >
            Create User
          </button>
        </div>
      </div>
      <div
        className="col-md-2"
      />
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-2"
      />
      <div
        className="col-md-8"
      />
      <div
        className="col-md-2"
      />
    </div>
  </form>
</div>
`;
